plugins/jira/lib/samson_jira/samson_plugin.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module SamsonJira
+  class Engine < Rails::Engine
+  end
+
+  def self.transition_jira_tickets(deploy, output)
+    return unless deploy.succeeded?
+    return unless url = ENV['JIRA_BASE_URL'].to_s[/(.*)\/[a-z]/, 1]
```

Comment:
ENV vars should be validated on class creation + write in logs in something is missing? To improve debuggability.

Reply:
not having user/token configured is a valid usecase, so I don't want to spam the logs ...
if they are not set the prefix/transation ui is not shown, so that case should not happen 🤞